### ISSUE TITLE
Car Keeps Bouncing  in turn by turn navigation

### ISSUE THREAD

**Issue by nick70007** [mapbox/mapbox-navigation-ios#3313]
My car keeps bouncing out of road lane.. Below is the screenshot for that. I am calling GPS Update location every 5 second for upadting my location. Is it due to calling of an api every 5 seconds?

![Screenshot 2021-09-01 at 5 08 22 PM](https://user-images.githubusercontent.com/52743988/131665193-98037379-3a87-456e-b07d-7fbb6c0f28d8.png)

**Comment by MaximAlien** [comment]
Hello @nick70007. What do you mean by bouncing? Is it shifted position of the `UserCourseView` or something else? Also what API you're calling every 5 seconds?

**Comment by nick70007** [comment]
yes it shifts position and i am calling API every 5 seconds which updates my current location

**Comment by nick70007** [comment]
And also the GPS Lags and turn by turn navigation gets stucked and due to that app gets crashed

**Comment by 1ec5** [comment]
> i am calling API every 5 seconds which updates my current location

Can you say more? The navigation SDK automatically listens for location updates from Core Navigation, so there should be no need to call any API yourself to determine the user’s location.
